src/Pages/home/Containers/HomeContainer.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import Header from "../Components/Header";
+import Box from "@material-ui/core/Box";
+import Title from "../Components/Title";
+
+const HomeContainer: React.FC = () => {
+  return (
+    <Box>
```

Comment:
I'm not sure if this scope is implemented Fragment tag (<> </>), Shoude it still works?